content/developer/reference/cmdline.rst

```diff
@@ -254,6 +254,11 @@ Emails
 
     Email address used as <FROM> when Odoo needs to send mails
 
+.. option:: --from-filter <address>
+
+    Define for which email address this SMTP configuration can be used (can be empty, a domain name or an entire email address).
+    If the sender do not match this filter, his email will be encapsulated.
```

Comment:
Great start, do you mind if we change a couple of things?

`Define which email address the SMTP configuration will apply to. This field can be empty, a domain name, or an entire email address. If the sender's email address does not match this set filter, then the email will not be sent. `

I made some grammatical changes and shifted some words around. Does this include everything you wanted to communicate and is it technically accurate?

Reply:
@std-odoo 

Can you review this update to the changes above (lines 259-260)? 👍 

``Define which email address the SMTP configuration will apply to. This field can be empty, a domain name, or an entire email address. If the sender's email address does not match this set filter, then the email will be encapsulated using a combination of the two system parameters: mail.default.from | mail.default.from.``

Reply:
maybe we can add an example of encapsulation ? like
```
"Admin" <admin@example.com> => "Admin" <notifications@mycompany.com>
```

otherwise LGTM :)